src/services/cronJobs/checkAndUpdateEndaomentProject.ts

```
@@ -0,0 +1,88 @@
+      logger.info(`Project ID ${project.id} marked as cancelled (offboarded).`);
+    } else {
+      // Update project fields with the fetched data
+      project.title = orgData.name || project.title;
+      project.description = orgData.description || project.description;
+      project.updatedAt = new Date();
+      logger.info(`Project ID ${project.id} updated successfully.`);
+    }
+
+    await project.save();
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)